commands/cardcreator.js

```diff
@@ -74,9 +74,17 @@ class CardCreator {
       return [];
     }
     const cardsDir = path.join(themesDir, defaultTheme, 'cards');
```

Comment:
nit: thoughts on renaming to `themeCardsDir`?

Reply:
honestly I don't want to make more differences between the two commands

Reply:
ok

Reply:
we originally made them two separate commands to keep them dumb and decoupled, but there have been literally 0 divergences between them in the past year, besides the different folder name

Reply:
I'll rename them both to themeCardDir though